app/assets/javascripts/answers.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener('ajax:success', function(event) {
```

Comment:
nope, this event should be attached only to your form for answer submitting. You don't want this handler to be invoked for EVERY form on the site, right?